src/remix.ts

```diff
@@ -0,0 +1,34 @@
+import type { AppLoadContext } from '@remix-run/cloudflare'
+import type { Context } from 'hono'
+
+export type GetLoadContext = (args: {
+  request: Request
+  context: {
+    // Relaxing the type definition
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    cloudflare: any
```

Comment:
It seems like we could do more than `any` here.

It might be tricky/impractical to use the "real" `Env` and `PlatformProxy` types, but I think this works

```suggestion
    cloudflare: {
      env: Record<string, unknown>
      cf: IncomingRequestCfProperties
      ctx: ExecutionContext
      caches: typeof caches
    }
```